boards/silabs/dev_kits/pg23_pk2504a/Kconfig.defconfig

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2025 Christoph Jans
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_PG23_PK2504A
+
+config LOG_BACKEND_SWO_FREQ_HZ
+	default 875000
+	depends on LOG_BACKEND_SWO
+
+endif # BOARD_PG23_PK2504A
```

Comment:
This comment is useful only if the associated if is not obvious (which is not the case here). Otherwise DRY (Don't Repeat Yourself) rule applies.